test/TransitivePathTest.cpp

```diff
@@ -1125,3 +1128,211 @@ INSTANTIATE_TEST_SUITE_P(
       result += std::get<1>(info.param) ? "Lazy" : "FullyMaterialized";
       return result;
     });
+
+// _____________________________________________________________________________
+TEST(TransitivePathBinSearch, successors) {
+  auto input =
+      makeIdTableFromVector({{0, 10}, {1, 10}, {2, 11}, {2, 12}, {3, 13}});
+  BinSearchMap binSearchMap{input.getColumn(0), input.getColumn(1)};
+  EXPECT_THAT(binSearchMap.successors(V(0)), ElementsAre(V(10)));
+  EXPECT_THAT(binSearchMap.successors(V(1)), ElementsAre(V(10)));
+  EXPECT_THAT(binSearchMap.successors(V(2)), ElementsAre(V(11), V(12)));
+  EXPECT_THAT(binSearchMap.successors(V(3)), ElementsAre(V(13)));
+}
+
+// _____________________________________________________________________________
+TEST(TransitivePathBinSearch, successorsWithGraph) {
+  auto input = makeIdTableFromVector(
+      {{0, 10, 100}, {1, 10, 100}, {2, 11, 100}, {2, 12, 101}, {3, 13, 101}});
+  BinSearchMap binSearchMap{input.getColumn(0), input.getColumn(1),
+                            input.getColumn(2)};
+  binSearchMap.setGraphId(V(100));
+  EXPECT_THAT(binSearchMap.successors(V(0)), ElementsAre(V(10)));
+  EXPECT_THAT(binSearchMap.successors(V(1)), ElementsAre(V(10)));
+  EXPECT_THAT(binSearchMap.successors(V(2)), ElementsAre(V(11)));
+  EXPECT_THAT(binSearchMap.successors(V(3)), ElementsAre());
+
+  binSearchMap.setGraphId(V(101));
+  EXPECT_THAT(binSearchMap.successors(V(0)), ElementsAre());
+  EXPECT_THAT(binSearchMap.successors(V(1)), ElementsAre());
+  EXPECT_THAT(binSearchMap.successors(V(2)), ElementsAre(V(12)));
+  EXPECT_THAT(binSearchMap.successors(V(3)), ElementsAre(V(13)));
```

Comment:
Are there tests missing for the `getEquivalent...`  for the binary search wrapper?